google/resource_google_project_iam_policy_test.go

```diff
@@ -594,6 +594,7 @@ func TestIamMergeBindings(t *testing.T) {
 						"member-2",
 					},
 				},
+				{Role: "empty-role", Members: []string{}},
```

Comment:
This seems like it needs to be formatted. Run go fmt before submitting

Reply:
Nope, `go fmt` thinks it's okay.  `go vet` would complain otherwise and Travis would fail, right?

Reply:
Yup. My bad. For some reason, I though a space would be required between `{` and `Role`.

You are good to go :)